### ISSUE TITLE
Add new plugin hook "admin_user_users_edit_overview_table" into admin/modules/user/users.php

### ISSUE THREAD

**Issue by SvePu** [mybb/mybb#4527]
To have possibility of adding new entries into acp user overview table, ~~move the plugins hook "admin_user_users_edit_overview" up~~ add a new plugin hook "admin_user_users_edit_overview_table" before outputting the table and make avatar cell row span variable.

![acp_overview_user](https://user-images.githubusercontent.com/9826631/155844517-02d4fc62-cda8-4009-92ab-e66ecc4c539d.png)

**Comment by Stefan-MyBB** [comment]
I understand the issue with that hook. However what happens to plugins using the hook right now?

**Comment by SvePu** [comment]
@Stefan-MyBB 
Atm you could use this hook to include a completely new table into this overview tab, but you can't add new rows into the existing table.

**Comment by Stefan-MyBB** [comment]
> @Stefan-MyBB Atm you could use this hook to include a completely new table into this overview tab

Moving the hook also moves that table.

**Comment by SvePu** [comment]
@Stefan-MyBB 
You are right, I didn't think about that..... maybe it's better to add a new hook.

**Comment by SvePu** [comment]
Added a new hook in https://github.com/mybb/mybb/pull/4528/commits/2c027259cc16bc30e20e9654c0269d1d55cb3fbf
